presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -114,6 +114,7 @@
     private boolean assumeCanonicalPartitionKeys;
 
     private boolean useOrcColumnNames;
+    private boolean orcBloomFiltersEnabled = true;
```

Comment:
I'm fine disabling it by default, however I'm not sure if ORC-101 has real implication in production environments as you'd expect this to be defined explicitly, and in some occasions the Presto cluster will run on the same cluster as HDFS/Hive meaning the default charset will be the same. Another reason to disable it by default for now is to let some people in the community run it in production for a few weeks and see if issues pop up. If the improvement is "stable" we can still enable it by default afterwards.